modules/dnn/include/opencv2/dnn/dnn.hpp

```diff
@@ -739,7 +739,7 @@ CV__DNN_INLINE_NS_BEGIN
 
         /** @brief Returns overall time for inference and timings (in ticks) for layers.
          * Indexes in returned vector correspond to layers ids. Some layers can be fused with others,
-         * in this case zero ticks count will be return for that skipped layers.
+         * in this case zero ticks count will be return for that skipped layers. Not supported by CUDA backend.
```

Comment:
I believe OpenCV/CPU backend only supports this method properly.
Other backends are asynchronous (like OpenCL, CUDA) or they don't provide per-layer information because tasks are scheduled as sub-graphs.

Reply:
I don't have that knowledge. It would be helpful to indicate limited backend support, so no one else wastes time to find out the hard way.

Reply:
It should be:

```
... DNN_BACKEND_OPENCV on DNN_TARGET_CPU only.
```

Reply:
Yes, I corrected it.